src/main/resources/assets/paths/draftDocumentIndex.yaml

```
@@ -0,0 +1,76 @@
+get:
+  summary: Get a draft
+  description: Provides the draft contents and metadata.
+  tags:
+  - Draft
+  parameters:
+    - name: draftUUID
+      in: path
+      description: The UUID identifying the draft
+      required: true
+      schema:
+        type: string
+        format: UUID v4.
+  responses:
+    200:
+     description: A draft with metadata.
+     content:
+       application/json:
+         schema:
+          $ref: '../schemas/DraftDetails.yaml'
+    401:
+      description: Unauthorized.  Did you remember to authenticate?
+    404:
+      description: The draft cannot be found.
+put:
+  summary: Updates a draft
+  description: Updates the contents of the draft.
+  tags:
+    - Draft
+  parameters:
+    - name: draftUUID
+      in: path
+      description: The UUID identifying the draft
+      required: true
+      schema:
+        type: string
+        format: UUID v4.
+  requestBody:
+    description: |
+      Schema validation is not performed and attachments associated with the draft may be uploaded after the draft is created.
+    content:
+      application/json:
+        schema:
+          type: object
+  responses:
+      200:
+        description: A draft with metadata.
+        content:
+          application/json:
+           schema:
+            $ref: '../schemas/DraftDetails.yaml'
+      401:
+        description: Unauthorized.  Did you remember to authenticate?
+      404:
+        description: The draft cannot be found.
+
+delete:
+  summary: Deletes a draft
+  description: Deletes the contents and attachments of a draft.
+  tags:
+    - Draft
+  parameters:
+    - name: draftUUID
+      in: path
+      description: The UUID identifying the draft
+      required: true
+      schema:
+        type: string
+        format: UUID v4.
```

```suggestion
        format: uuid
```

I think this is the only valid OpenAPI UUID format.

---

From the [Swagger docs](https://swagger.io/docs/specification/v3_0/data-models/data-types/#:~:text=pattern%20is%20specified.-,String%20Formats,-An%20optional%20format), format is open and only "serves as a hint at the contents and format of the string."

I'm fine with updating it to uuid or UUID v4 throughout.  After having worked with different formats of UUIDs, I have a greater appreciation for hinting more specifics.

Let me know what you think!

---

@fboulnois - I think I came up with something that can satisfy both of us for this one.  I updated the format to uuid and added the (updated, thank you) link to the description so it's visible to anyone that might read the documentation.